gpii/node_modules/gpii-oauth2/gpii-oauth2-authz-server/src/AuthorizationService.js

```diff
@@ -67,6 +81,24 @@ fluid.defaults("gpii.oauth2.authorizationService", {
         getAuthForAccessToken: {
             func: "{dataStore}.findAuthByAccessToken"
                 // accessToken
+        },
+        getClientByClientCredentialsAccessToken: {
+            func: "{dataStore}.findClientByClientCredentialsAccessToken"
+                // accessToken
+        },
+        getClientCredentialsTokenPrivs: {
+            func: "{dataStore}.findClientCredentialsTokenPrivs"
+                // accessToken
+        },
+        grantClientCredentialsAccessToken: {
+            funcName: "gpii.oauth2.authorizationService.grantClientCredentialsAccessToken",
+            args: ["{dataStore}", "{arguments}.0", "{arguments}.1"]
+                // clientId, scope
+        },
+        savePrefs: {
+            funcName: "gpii.oauth2.authorizationService.savePrefs",
+            args: ["{preferencesDataSource}", "{arguments}.0", "{arguments}.1", "{arguments}.2"]
+                // preferences, view, callback function
```

Comment:
This invoker record and comment no longer match the implementing function (as we are now returning the promise rather than using a callback).